build/package.json

```diff
@@ -23,7 +23,7 @@
     "raw-loader": "^0.5.1",
     "webdriver": "0.0.1",
     "webdriver-manager": "7.0.1",
-    "webdrivercss": "^1.1.4",
+    "webdrivercss": "cyaninc/webdrivercss",
```

Comment:
Let's make a tag for the version we are currently at in our fork, name it the version that we are mirroring from `webdrivercss`, for instance, if we are patching version `1.1.5` of `webdrivercss` let's make a tag called `1.1.5` and then this line becomes:

```
"webdrivercss": "cyaninc/webdrivercss#1.1.5",
```